lib/tests/streamlit/runtime/app_session_test.py

```diff
@@ -543,8 +543,15 @@ def test_disconnect_file_watchers_removes_refs(self):
         self.assertGreater(len(gc.get_referrers(session)), 0)
 
         session.disconnect_file_watchers()
-        # Ensure that we don't count refs to session from an object that would have been
-        # garbage collected along with it.
+
+        # Run the gc to ensure that we don't count refs to session from an object that
```

Comment:
Would it be possible to leave a GitHub link to an example of a failure just in case for the future?

Reply:
actions logs are only retained for a set amount of time before being deleted, so not really